src/Snapshot/Command/ScheduleSnapshot.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * This file is part of the broadway/snapshotting package.
+ *
+ *  (c) Qandidate.com <opensource@qandidate.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Broadway\Snapshotting\Snapshot\Command;
+
+class ScheduleSnapshot
+{
+    /**
+     * @var string
+     */
+    private $aggregateRootId;
+
+    /**
+     * @param string $aggregateRootId
+     */
+    public function __construct($aggregateRootId)
```

Comment:
What are your thoughts on adding the playhead you want to snapshot at here?
Wouldn't it be safer (at least more explicit) to know the playhead on the aggregate hasn't changed in between?

Reply:
Turns out that I did miss out on the aggregate class name, leaving you without any way of reasonably knowing what aggregate you're dealing with. This has now been added to the `ScheduleSnapshot` command, allowing for any of the `AggregateRootFactory` implementations to be used to create a new aggregate to apply events to.